browser/create-template/templates/nextjs-site/src/views/MenuItem/MenuItem.tsx

```diff
@@ -0,0 +1,39 @@
+import type { MenuItem } from '@/ontologies/website';
+import MenuItemLink from './MenuItemLink';
+import styles from './MenuItem.module.css';
+import { store } from '@/app/store';
+import { unknownSubject } from '@tomic/lib';
+
+const MenuItem = async ({ subject }: { subject: string }) => {
+  const menuItem = await store.getResource<MenuItem>(subject ?? unknownSubject);
+
+  return menuItem.props.subItems && menuItem.props.subItems.length > 0 ? (
+    <>
+      <button
+        className={styles.button}
+        popoverTarget='popOver'
```

Comment:
polyfill: https://github.com/oddbird/css-anchor-positioning

Reply:
This polyfill does not work. Check `MenuItem.svelte` for a custom solution